test/integration/smoke/test_privategw_acl.py

```diff
@@ -720,7 +720,7 @@ def check_pvt_gw_connectivity(self, virtual_machine, public_ip, vms_ips):
         succeeded_pings = 0
         minimum_vms_to_pass = 2
         for vm_ip in vms_ips:
-            ssh_command = "ping -c 5 %s" % vm_ip
+            ssh_command = "ping -c 10 %s" % vm_ip
```

Comment:
@ernjvr is this related change? if it's a bugfix can we have it in a separate PR?